test/powershell/Language/Scripting/Debugging/Debugging.Tests.ps1

```diff
@@ -0,0 +1,96 @@
+$script1 = @'
+'aaa'.ToString() > $null
+'aa' > $null
+"a" 2> $null | % { $_ }
+'bb' > $null
+'bb'.ToSTring() > $null
+'bbb'
+'@
+$script2 = @'
+"line 1"
+"line 2"
+"line 3"
+'@
+
+$root = git rev-parse --show-toplevel
```

Comment:
Are you aware that we already have tests which rely on git being installed? Are you suggesting that those then should be changed?